dss-api.yml

```diff
@@ -1111,28 +1098,6 @@ definitions:
           - name
           - content-type
           - indexed
-  file_metadata:
```

Comment:
Is file metadata associated with the backlink (file -> bundle)?

Reply:
No, this is just an orphaned section in the Swagger file. I committed the removal as a separate commit.

Reply:
If this yml section is really orphaned/unneeded than it LGTM.

Reply:
I think once upon a time, I wanted this to be the return value of GET /files/<version>.  However, that data ended up being returned in headers, so this is entirely obsolete.

I would inline blob_common_metadata into file_version, since that's the only usage of it.